STAGES.md

```diff
@@ -90,8 +89,7 @@ formally rejected by a [W3C Working Group].
 
 ### Recognized Browser Vendors
 
-Recognized browser vendors include, in alphabetical order; Apple, Google,
-Microsoft, Mozilla, and Opera.
+Recognized browser vendors include, in alphabetical order; Apple, Google and Mozilla.
```

Comment:
```suggestion
Recognized browser vendors include, in alphabetical order; Apple (Safari/Webkit), Google (Chrome/Chromium) and Mozilla (Firefox/Gecko).
```

Listing the actual browsers and engines is useful I think.

Reply:
Completely agree ❤️